test/persistence-hooks.suite.js

```diff
@@ -1202,9 +1227,14 @@ module.exports = function(dataSource, should) {
       });
     });
 
-    function pushContextAndNext() {
+    function pushContextAndNext(fn) {
       return function(context, next) {
+        if (typeof fn === 'function') {
+          fn(context);
+        }
+
         context = deepCloneToObject(context);
+        context.hookState.test = true;
```

Comment:
This is very convoluted way of testing this feature. The other tests for Operations hook strive to make it very clear what is being tested (what features are supported) and also to make each test as small and isolated as feasible.